config/routes.rb

```diff
@@ -30,8 +30,7 @@
     resources :steps, controller: :user_steps, path: 'krok'
     resources :tasks, controller: :user_tasks, path: 'ulohy' do
       member do
-        post :complete
-        post :undo
+        post :toggle_complete
```

Comment:
Toto tu inak bolo takto naschval. Zober si situaciu ze to nahodou mas v dvoch taboch a v jednom kliknes ze toggle a v druhom potom tiez toggle. vysledkom bude ze to bude odznacene. operacie by mali byt idempotentne (t.j. ked ich zavolas viac krat za sebou spravia to iste)

Reply:
Updatol som to, su to 2 akcie opat. Pozri na to, ci to tak moze byt, daval som tam case.